Add database migration from version 10 to 13 (#969)
uyuni-project/uyuni-docs: modules/upgrade/pages/db-migration-13.adoc

> . Start the upgrade:

Reviewer: ```suggestion
. Start the upgrade.
  If you have version 12, run:
```

+
----

> if you have version 12, run:

Reviewer: ```suggestion
```

/usr/lib/susemanager/bin/pg-migrate-12-to-13.sh

> if you have version 10, run:

Reviewer: ```suggestion
----
+
  If you have version 10, run:
+
----
```

/usr/lib/susemanager/bin/pg-migrate-10-to-13.sh
----
. When the upgrade has successfully completed, you can safely delete the old database directory and reclaim lost disk space.

>   The old directory is renamed to [path]``/var/lib/pgsql/data-pg12`` or [path]``/var/lib/pgsql/data-pg10``, depending on whether you had version 12 or 10..

Reviewer: ```suggestion
  The old directory is renamed to [path]``/var/lib/pgsql/data-pg12`` or [path]``/var/lib/pgsql/data-pg10``, depending on the version you started from.
```

> . Start the upgrade:

Reviewer: ```suggestion
. Start the upgrade.
  If you are migrating from version 12:
```

+
----

> If you are migrating from version 12:

Reviewer: ```suggestion
```

/usr/lib/susemanager/bin/pg-migrate-12-to-13.sh fast

> If you are migrating from version 10:

Reviewer: ```suggestion
----
+ 
  If you are migrating from version 10:
+
----
```

/usr/lib/susemanager/bin/pg-migrate-10-to-13.sh fast
----
. When the upgrade has successfully completed, you can safely delete the old database directory and reclaim lost disk space.

>   The old directory is renamed to [path]``/var/lib/pgsql/data-pg12`` or [path]``/var/lib/pgsql/data-pg10``, depending on whether you had version 12 or 10..

Reviewer: ```suggestion
  The old directory is renamed to [path]``/var/lib/pgsql/data-pg12`` or [path]``/var/lib/pgsql/data-pg10``, depending on the version you started from..
```
